apps/experiments/views/experiment.py

```diff
@@ -1079,6 +1079,10 @@ def experiment_session_messages_view(request, team_slug: str, experiment_id: uui
     selected_tags = list(filter(None, request.GET.get("tag_filter", "").split(",")))
     language = request.GET.get("language", "")
     show_original_translation = request.GET.get("show_original_translation") == "on" and language
+    try:
+        message_id = int(request.GET.get("message_id", 0))
```

Comment:
Since you are already doing error handling, there's no need to provide a default value. Having the default value is a bit misleading since it makes it seem like you want to always highlight a message.
```suggestion
        message_id = int(request.GET.get("message_id"))
```